File: crud_filmes/importacao_automatica/robo_python/classes/Repositorios/FilmeRepo.py
from classes.bd.Conexao import Conexao

class FilmeRepo:
    xsql = ''

    def __init__(self):
        self.xsql = Conexao()
    
    def buscarFilme(self,nomeFilme):
        try:
            sql = "SELECT * FROM filmes WHERE nome = '" + nomeFilme + "'"
            retorno = self.xsql.requisitar(sql)
            return retorno 
        except Exception as e:
            print(e)
            return False
    def inserirFilme(self,parametros):
        try:
            sql = "INSERT INTO filmes(nome, data_lancamento, tipo_filme, diretor, ator_principal, detalhes, imagem_b64) Values (" + parametros + ");"
            retorno = self.xsql.inserir(sql)
            return True
        except Exception as e:
            print(e)
            return False

File: crud_filmes/classe/repositorios/FilmeRepo.php
<?php
require_once('classe/xsql.php');

class FilmeRepo{
    protected $conexao;
    function __construct()
    {
       $this->conexao = new Xsql();     
    }


    /**
     * 
     */
    function pegardetalhesFilme ($filme) {
        try {
            $sql = "
            SELECT
            A.nome,
            data_lancamento,
            (SELECT x.descricao FROM tipo_filme x WHERE x.codigo = A.tipo_filme) AS tipo_filme,
            (SELECT x.nome FROM pessoas x WHERE x.codigo = A.ator_principal) AS ator_principal,
            A.detalhes
        FROM 
            filmes A
        WHERE 
            A.codigo = $filme";
            $retorno = $this->conexao->requisitar($sql);
            if ($retorno === false) {
                throw new Exception($this->conexao->mensagem);
            }
            return $retorno;
        }
        catch (Exception $e) {
            trigger_error($e->getMessage());
            return false;
        }
    }
}